docs/Documentation/Scripting/Building-Blocks/Integration-List.md

```diff
@@ -226,6 +226,45 @@ This means that `enter` is not completed when the player gets the objective and
     npcrange 3,5 enter 20 events:master_inRange
     ```
 
+### Variables
+
+#### Citizen Variable: `%citizen.<id>.<type>%`
```

Comment:
I think we should use `argument` or something similar instead of type. My first impression was the npc type which is not meant here.

Reply:
This has been fixed.